.github/workflows/build.yaml

```diff
@@ -13,6 +13,17 @@ on:
         required: true
         type: boolean
         default: false
+      refLTITag:
+        description: 'Reference LTI Tag'
+        type: string
+        required: true
+        default: main
+      # Specifying the default value as 'Liberty-tools-intellij' because we require the forked repository path for a successful build.
```

Comment:
"because we require the forked repository path for a successful build." what does this comment mean?

Reply:
I removed this comment from here , I already mentioned the need of this parameter in cronjob.yaml file line no `84`